lib/chef/property.rb

```diff
@@ -305,7 +305,7 @@ def has_default?
     #
     def required?(action = nil)
       if !action.nil? && options[:required].is_a?(Array)
-        options[:required].include?(action)
+        (options[:required] & Array(action)).any?
```

Comment:
How does this functionally differ from the original check? When running through some scenarios, I can't come up with one that returns a different result given pre-conditions of  a non-nil action and :required having an Array.

Reply:
> How does this functionally differ from the original check?

The original cannot handle a check for multiple actions. It's fine if it's checking for just :create for example, but if `action` is an array with both :create and :delete it would fail with a simple `include?` test. I wrote this over a year ago and the details of the context now escape me, but IIRC there are situations where this needs to test for multiple actions—where `action` is going to be an array of actions, and `include?` only tests for the presence of a single object.